devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -90,46 +163,55 @@ def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null) {
 	log.debug "Basic ${cmd}" + (ep ? " from endpoint $ep" : "")
 	def value = cmd.value ? "on" : "off"
-	ep ? changeSwitch(ep, value) : []
+
+	if (isZoozZenStripV2()) {
+		// device sends also reports without any endpoint specified, therefore all endpoins must be querried
+		// sometimes it also reports 0.0 Wattage only until it's queried for it, then it starts reporting real values
```

Comment:
nit (spelling):

// device also sends reports without any endpoint specified, therefore all endpoints must be queried
// sometimes it also reports 0.0 Wattage only until it's queried for it, then it starts reporting real values

Reply:
fixed